src/content/docs/zh-cn/guides/view-transitions.mdx

```diff
@@ -233,18 +233,82 @@ const myFade = {
 
 <header transition:animate={myFade}> ... </header>
 ```
+## 路由控制
 
-## 阻止客户端导航
+`<ViewTransitions />` 路由通过监听以下事件来处理导航：
 
-一旦你在页面上添加了 `<ViewTransitions />` 组件，就可以通过客户端路由导航到页面上链接到站点上的另一个页面的每个锚点。有一些情况下，你可能不希望通过 CSR 导航。一个例子是，如果链接指向任何非页面，例如 `public/` 文件夹中的 PDF。或者是一个生成图像的 API 路由。
+- 点击 `<a>` 元素。
+- 后退和前进导航事件。
 
-在这些情况下，你可以使用 `data-astro-reload` 属性来选择性地阻止每个链接使用客户端路由，如下所示：
+如下选项允许你进一步控制在路由中发生的导航事件：
 
-```html
-<a href="/quarterly-earnings.pdf" data-astro-reload>
+-  `astro-data-reload`：一个 `<a>` 标签属性，用于 [进行强制全页面导航](#阻止客户端导航)。
+-  `astro-data-history="auto | push | replace"`：一个 `<a>` 标签属性，用于 [控制浏览器的历史记录](#替换浏览器历史条目)。
+-  `navigate(href, options)`：任何客户端脚本或客户端组件可用的方法，用于[触发导航](#阻止客户端导航)。
+
+### 阻止客户端导航
+
+由于关联的两个页面须使用 `<ViewTransitions />` 路由以防止重新加载全页面，因此在某些情况下你无法通过 CSR 进行导航。你可能也不希望在每次导航更改时都使用客户端路由，而是更希望在特定路由上进行传统的页面导航。
+
+你可以通过将 `data-astro-reload` 属性添加到任何 `<a>` 标签中，按链接的方式选择退出客户端路由。此属性将覆盖任何现有的 `<ViewTransitions />` 组件，而是在导航期间触发浏览器刷新。
+
+以下示例显示了仅从主页导航到文章时如何阻止客户端路由。这样，当从文章列表页面导航到同一页面时，仍然可以对共享元素（例如英雄图像）进行动画处理：
```

Comment:
> This still allows you to have animation on shared elements, such as a hero image, when navigating to the same page from an article listing page:

Hero image 直译成“英雄图像”不太合理，类似 “superman image” 的感觉意思就完全变了。

> (web design) An eye-catching, prominent image, used as an over-sized (almost background-like) header. -- [wiktionary](https://en.wiktionary.org/wiki/hero_image)

> The term “hero image” in web design refers to a large, attention-grabbing picture with text typically shown in the above-the-fold area of the webpage, directly beneath the website header. -- [vwo.com](https://vwo.com/glossary/hero-image/)

> The term “hero” was first used in the world of theater, and it was the prop (it’s usually called the [hero prop](https://en.wikipedia.org/wiki/Theatrical_property)) that is more detailed than usual because it’s meant to be seen or viewed in close range. -- [foxxr.com](https://foxxr.com/blog/epic-story-of-the-hero-image/)

我觉得可以翻译成“主视觉图”，不知道有没有其他更合理的翻译。

Hero 之后可以加到词汇表里。

```suggestion
以下示例显示了仅从主页导航到文章时如何阻止客户端路由。这样，当从文章列表页面导航到同一页面时，仍然可以对共享元素（例如主视觉图）进行动画处理：
```